ibm/mas_devops/roles/appconnect_install/templates/appconnectcfg.yml.j2

```diff
@@ -0,0 +1,15 @@
+# AppConnect system configuration (addons.mas.ibm.com/v1)
+---
+apiVersion: addons.mas.ibm.com/v1
+kind: AppConnect
+metadata:
+  name: "{{mas_instance_id}}-addons-appconnect"
+  namespace: "mas-{{mas_instance_id}}-core"
+  labels:
+    mas.ibm.com/configScope: system
+    mas.ibm.com/instanceId: "{{mas_instance_id}}"
+spec:
+  displayName: MAS AppConnect Configuration
+  config:
+    dashboard: "{{ appconnect_url }}"
+    users: []
```

Comment:
Please add a newline here for linting purposes (even though atm, the j2 prefix prevents current linting doing this file, it won't in the future).